regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -56,18 +55,22 @@ var CommentView = Backbone.View.extend({
     });
 
     this.listenTo(CommentEvents, 'comment:target', this.target);
-    this.listenTo(CommentEvents, 'comment:update', this.update);
 
-    if (this.section) {
-      this.load();
-    }
+    this.setSection(options.section);
   },
 
-  render: function() {},
-
-  setSection: function(section) {
-    this.section = section;
-    this.key = 'comment:' + section;
+  setSection: function(section, overwrite) {
+    if (this.model) {
+      this.stopListening(this.model);
+    }
+    if (section) {
+      this.model = overwrite ?
+        new CommentModel({id: section}) :
+        comments.get(section) || new CommentModel({id: section});
+      this.listenTo(this.model, 'destroy', this.setSection.bind(this, section, true));
```

Comment:
The overwrite flag confuses me -- it seems that sometimes we allow `this.model` to be null and other times we want it to be a "blank" model. What's the distinction?